apps/web/src/features/touchpoints/components/TouchpointCreateForm.tsx

```
@@ -0,0 +1,545 @@
+								required
+								error={form.formState.errors.student_id?.message}
+							>
+								<Popover
+									open={studentSearchOpen}
+									onOpenChange={setStudentSearchOpen}
+								>
+									<PopoverTrigger asChild>
+										<Button
+											variant="outline"
+											role="combobox"
+											aria-expanded={studentSearchOpen}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)